### PR TITLE
added back script, arranged Important under it

### PR DESCRIPTION
**Pull request by twblackwell**

### CODE REVIEW

File: frontend/src/components/Bill/Bill.jsx [hunk 1]
```diff
@@ -4,7 +4,21 @@ import { useParams } from 'react-router-dom';
...
+
+  } = billData;
+  return (
+    1 +
+    items[activeVersion].coSponsors.size +
+    items[activeVersion].multiSponsors.size
+  );
+}
+
+function getSponsorNumber(billData) { // TODO: change to getSponsors
```
Comment (elliott-king) [inline frontend/src/components/Bill/Bill.jsx:21]:
```suggestion
function getSponsorNumber(billData) {
```
Should this comment be here?

File: frontend/src/components/Bill/Bill.jsx [hunk 2]
```diff
@@ -4,7 +4,21 @@ import { useParams } from 'react-router-dom';
 import LocForm from "./LocForm";
 import './Bill.css';
 
-function getSponsorNumber(billData) {
+function getSponsors(billData) { // TODO: change to getSponsors
```
Comment (elliott-king) [inline frontend/src/components/Bill/Bill.jsx:7]:
```suggestion
function getSponsors(billData) {
```

Should this comment be here?

File: frontend/src/components/Bill/Bill.jsx [hunk 3]
```diff
@@ -141,19 +155,17 @@ export const Bill = () => {
...
       <div className="summary">
-        <h2>{title}</h2>
+        <h2>{printNo}: {title}</h2>
         <p>{summary}</p>
-        <div className="category">
-          {bill.category} <p>(Sample Category)</p>
-        </div>
+        {/* <div className="category">
+           <p>(Sample Category)</p>
+        </div> */}
```
Comment (elliott-king) [inline frontend/src/components/Bill/Bill.jsx:162]: Remove comment or add context

```suggestion
```

File: frontend/src/components/Bill/LocForm.jsx [hunk 4]
```diff
@@ -131,10 +127,10 @@ const LocForm = ({ sponsorNames }) => {
           {Object.entries(representatives).map(([name, rep]) => (
             <div key={rep.name} className="representative">
               <h3>
-                {rep.title} {rep.name}:{' '}
-                {sponsorshipStatus[rep.name]
+                {rep.title} {rep.name}
+                {/* {sponsorshipStatus[rep.name]
                   ? 'Already a sponsor'
-                  : 'NOT A SPONSOR'}
+                  : 'NOT A SPONSOR'} */}
```
Comment (elliott-king) [inline frontend/src/components/Bill/LocForm.jsx:133]: Remove comment or add context to why it's here.

File: frontend/src/components/Bill/Bill.jsx [hunk 5]
```diff
@@ -162,21 +174,28 @@ export const Bill = () => {
...
-          <h4>Why is this important? Why should this bill pass?</h4>
+      </div>
+      
+      <div className="action">
+        <h2>Take Action!</h2>
+        <LocForm sponsorNames={getSponsorNames(bill)}/>
+        <div className="script">
+          <h2>Script when calling a Legislator</h2>
+          <p>
+            Hi, my name is [Your Name] and I'm a constituent from {window.sessionStorage.getItem('placeName')}. I'm calling to urge [Legislator's Name] to support {printNo}.
```
Comment (elliott-king) [inline frontend/src/components/Bill/Bill.jsx:185]:
```suggestion
            Hi, my name is [Your Name] and I'm a constituent from {window.sessionStorage.getItem('placeName') || '[Borough]'}. I'm calling to urge [Legislator's Name] to support {printNo}.
```


This shows nothing if the user hasn't filled it out, so I'm suggesting a default value

**Review by elliott-king — CHANGES_REQUESTED** [review]

File: frontend/src/components/Bill/LocForm.jsx (hunk 4 above)
Comment (twblackwell) [reply frontend/src/components/Bill/LocForm.jsx:133]: I'll add context for which representatives the bill page shows and whether or not they are already a co-Sponsor

**Pull request closed without merging**